src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj

```diff
@@ -121,6 +121,21 @@
     <NativeRuntimeAsset Include="$(_diaSymReaderPathIfExists)" />
   </ItemGroup>
 
+  <!-- VS uses this file to show the target framework in the drop down. -->
+  <Target Name="CreateDotVersionFile" DependsOnTargets="InitializeSourceControlInformationFromSourceControlManager" BeforeTargets="GetFilesToPublish">
```

Comment:
Is this not needed all up? aka, will this fit better on the SFX sdk. Also, this feels like a really soft contract with VS. Why do they need said file? And can this be tested for, or added as a check in the SFX sdk?

Reply:
@dsplaisted can speak more to the implicit contract with VS and if we need to do this all up in the SFX SDK.

Reply:
I don't know the details, it has to do with how the template engine works.  @phenning or someone from @donJoseLuis's team can probably explain more about it.

It does seem like it should go in the shared framework SDK, as it looks like all 3 shared frameworks in .NET 5 had the .version file.  But if we want to fix this for .NET 6 preview 1 then we might want a quicker change in this repo for that.